packages/banner/src/Banner.tsx

```diff
@@ -0,0 +1,234 @@
+import React from 'react';
+import ImportantWithCircleIcon from '@leafygreen-ui/icon/dist/ImportantWithCircle';
+import InfoWithCircleIcon from '@leafygreen-ui/icon/dist/InfoWithCircle';
+import WarningIcon from '@leafygreen-ui/icon/dist/Warning';
+import CheckmarkWithCircleIcon from '@leafygreen-ui/icon/dist/CheckmarkWithCircle';
+import XIcon from '@leafygreen-ui/icon/dist/X';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const Variant = {
+  Info: 'info',
+  Warning: 'warning',
+  Danger: 'danger',
+  Success: 'success',
+} as const;
+
+type Variant = typeof Variant[keyof typeof Variant];
+
+export { Variant };
+
+const baseBannerStyles = css`
+  position: relative;
+  display: flex;
+  min-height: 40px;
+  padding: 9px 12px 9px 20px;
+  border-width: 1px;
+  border-style: solid;
+  border-radius: 6px;
+  font-size: 14px;
+  line-height: 20px;
+
+  &:before {
```

Comment:
Is there a reason we're using a pseudo-class for this rather than a border-left? You're already controlling the border-left's color independently.

Reply:
Great point, ty.